test/check-api

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/python3
 
-import composertest
+from composertest import ComposerTestCase
+from cockpittest import main
```

Comment:
This import style makes it hard for me to read and review code, because it's not immediately apparent that a variable is not local to the file. I also have to jump up to the top of the file to find out which module a definition is coming from. Especially for cases like the one further down, which is just called `TestCase` and could come from `cockpittest` or `unittest`.

Reply:
Understood. In longer files I'd agree. These are really short and easy to see.